Tests/Foundation/TestTimeZone.swift

```diff
@@ -164,7 +164,7 @@ class TestTimeZone: XCTestCase {
         var lt = tm()
         localtime_r(&t, &lt)
         let zoneName = NSTimeZone.system.abbreviation() ?? "Invalid Abbreviation"
-        let expectedName = String(cString: lt.tm_zone, encoding: .ascii) ?? "Invalid Zone"
+        let expectedName = String(cString: lt.tm_zone!, encoding: .ascii) ?? "Invalid Zone"
```

Comment:
Does this force unwrap produce a warning for non-Android platforms now?

Reply:
I took a look at the [linux](https://ci.swift.org/job/swift-corelibs-foundation-PR-Linux/683/consoleText) and [Windows](https://ci-external.swift.org/job/swift-corelibs-foundation-PR-windows/589/consoleText) CI output for this pull, and a search for `testtimezone` only turns up these tests running, no warnings.